include/seqan3/search/detail/policy_result_builder.hpp

```diff
@@ -39,14 +40,22 @@ struct policy_result_builder
     //!\endcond
     auto make_results(std::vector<index_cursor_t> internal_hits, configuration_t const &)
     {
-        return internal_hits;
+        using search_result_t = search_result<size_t, index_cursor_t, size_t, size_t>;
+        std::vector<search_result_t> results;
+        results.reserve(internal_hits.size());
+
+        for (auto const & cursor : internal_hits)
+            results.emplace_back(0, cursor);
```

Comment:
This is the problem:

You try to construct `search_result_t` within the vector, but the vector is not allowed to do this. In the context of the vector, the constructor is private. Hence the error.